### ISSUE TITLE
Requesting Removal of Update Music

### ISSUE THREAD

**Issue by dangero2000** [maxe-hsieh/subtitle_reader#33]
In installers with background music, typically you can use the NVDA audio ducking feature to turn the music down so you can hear your speech over the music. In this case, since the music is coming through NVDA directly, you cannot duck the audio or turn it down. This means that, when the changelog pops up, I cannot review it with my translator or anything like that. I have to quickly tab to the update button and press it before the music gets loud enough to drown everything out, except in cases where the music is piano music or something similar. I believe it would be best if the update music was removed.

**Comment by maxe-hsieh** [comment]
I will try to separate the updater in the next version to support NVDA's audio ducking feature.
You can also go to NVDA -> Tools -> Subtitle Reader -> Open Change Log to read the log.
Playing different music for each version during updates is my motivation for programming, so I probably won't remove this feature.
